OpenRA.Mods.Common/Traits/BotModules/UnitBuilderBotModule.cs

```diff
@@ -41,7 +41,7 @@ public class UnitBuilderBotModuleInfo : ConditionalTraitInfo
 
 	public class UnitBuilderBotModule : ConditionalTrait<UnitBuilderBotModuleInfo>, IBotTick, IBotNotifyIdleBaseUnits, IBotRequestUnitProduction, IGameSaveTraitData
 	{
-		public const int FeedbackTime = 30; // ticks; = a bit over 1s. must be >= netlag.
+		public const int FeedbackTime = 30;
```

Comment:
I think its ok/good to mention these are ticks and not ms.

```suggestion
		/// <summary>
		/// Feedback time in ticks.
		/// </summary>
		public const int FeedbackTime = 30;
```

Reply:
fixed